components/x-live-blog-wrapper/src/__tests__/post-tracker.test.js

```diff
@@ -28,22 +28,17 @@ describe('Live blog Visibility Tracker', function () {
 
 	it('should emit an error when a query is not passed to config', function () {
 		setupIntersectionObserverMock()
-		global.window = doc
 		global.window.addEventListener = jest.fn(() => {})
-		global.document = doc.window
```

Comment:
**question**: why have these been removed? does this work?

Reply:
ESLint _was_ complaining but when I just added it back in, it no longer is. The tests still passed but I have recovered these lines as I would rather not do the change if it is unnecessary.

Reply:
Turns out it was Jest that had an issue with modifying the global window, not ESLint. I have had to remove this to get tests to pass.